routes/votings/list-of-votings-f.js

```diff
@@ -1,6 +1,8 @@
 const knex = require("../../knex");
 
 function createList(list){
+    console.log("preslodo funkcie")
+    console.log(list)

```

Comment:
I guess you don't want to keep these comments here?

Reply:
there are not such comments or logs in the file